### DISCUSSION TITLE
Working ReadableStream example for larger files and upload using XMLHttpRequest

### DISCUSSION THREAD

**Discussion by bramevo** [openpgpjs/openpgpjs#1222] [Q&A]
I'm trying to figure out how to use openpgpjs for larger files. This requires ReadableStream but it's hard to figure out how this would work with something like XMLHttpRequest or another way to use/save the output.

Is it possible to use FileApi Blob.stream() and pass that to OpenPGPJS? Tried this multiple times but it always returns a ReadableStream {locked: false}

Should I use something like https://github.com/jimmywarting/StreamSaver.js to save the Stream to a file or could I push the stream to something useful?

**Comment by NickDarvey** [comment]
@bramevo, what did you end up doing?
